backend/src/main/java/clf/integra/backend/repository/IUserRepository.java

```diff
@@ -0,0 +1,22 @@
+package clf.integra.backend.repository;
+
+import clf.integra.backend.model.User;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface IUserRepository {
```

Comment:
1. Interface names do not usually start with I in Java, you just name the classes with `Impl` at the end
2. I think you did a bit too much, please sync with @lucianpopescu688